### ISSUE TITLE
Unable to take out Silenced beretta

### ISSUE THREAD

**Issue by magemagicc** [syl0r/Perpheads-Issues#318]
Upon trying to take out an m9 beretta I receive some script errors. There is no specific factor in this as it has occurred now for the past 3 days every time I've tried to take it out so I think it simply might be a bug  within my storage. Here are the errors I recieve

http://puu.sh/pVRA4/f3c39ae0d6.png

**Comment by StephenPuffs** [comment]
I've let Fredy know about this, he will be looking into it.

**Comment by syl0r** [comment]
is this still happening?

**Comment by magemagicc** [comment]
Yep still happens, now I got another silenced beretta here is what happens.

The inventory displays 2 seperate berettas both with their silencers on, upon taking one out, the image of both stays there but another image appears which is just an empty square with a '1' symboling there is 1 of this in my storage. Whenever I take one out, it will always take out the 'working' beretta, so I placed it in org storage and then I had apparently 3 still in my storage, tried clicking all of them and the empty picture one and just got the same error as before. If I place the 'working' beretta back in my storage it just places it as a seperate item, this can lead to me having something like this basically. http://puu.sh/q3Ir8/d3f52c6440.png In reality i only have 2 berettas with a silencer attatchement in my storage
